bedrock/settings/base.py

```diff
@@ -453,10 +453,15 @@ def JINJA_CONFIG():
 # This can be changed to use session once we do add a database.
 MESSAGE_STORAGE = 'django.contrib.messages.storage.cookie.CookieStorage'
 
-EMAIL_BACKEND = config(
-    'EMAIL_BACKEND',
-    default='django.core.mail.backends.console.EmailBackend' if DEBUG else
+
+def lazy_email_backend():
```

Comment:
Once we're done eliminating local.py I don't really see a point in keeping settings as a dir anyway-- I'd rather just have settings.py pull everything from env vars.